README.md

```diff
@@ -25,6 +25,36 @@ java -jar cryptomator-cli-x.y.z.jar \
 # you can now mount http://localhost:8080/demoVault/
 ```
 
+## Using as a docker image
+
+*Bridge networking with port forward:*
```

Comment:
Without disabling icc anyone can access that port.

Reply:
Yes, but this scenario is just for testing and understanding how the container works.
I use this specifically in a private network where another container running rclone pulls the vault from the cloud, cryptomator decrypts the vault and serves it to another container which grabs the content via WebDAV. Here one can define the --bind address to the container IP itself which the other container would access it on in a private network environment.
I will also update the README.md to indicate this.